diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/LineLength.kt

```diff
@@ -154,13 +170,19 @@ class LineLength(configRules: List<RulesConfig>) : DiktatRule(
             // can't fix this case
             return LongLineFixableCases.None
         }
+        // check, that space to split is a part of text - not code
```

Comment:
Please add a comment, why we can't fix it if it is code. Btw, shouldn't we treat it as an expression in completely separate branch of this rule?

Reply:
I added comment. No, because if code inside string is longer than expected, it will fixed. I added tests for this case.